src/App.test.tsx

```diff
@@ -4,14 +4,24 @@ import * as ReactDOM from "react-dom";
 import { shallow } from "enzyme";
 
 import App from "./App";
+import { OnChange } from "./Editor";
 
 it("renders without crashing", () => {
     const div = document.createElement("div");
     ReactDOM.render(<App />, div);
     ReactDOM.unmountComponentAtNode(div);
 });
 
-it("contains the text 'Test'", () => {
+it("contains the editor component", () => {
     const app = shallow(<App />);
-    expect(app.text()).toEqual("Test");
+    expect(app.children().length).toEqual(1);
+    expect(app.childAt(0).name()).toEqual("Editor");
+});
```

Comment:
I think it's better to use `expect(app.containsMatchingElement(<Editor/>)).toBeTruthy()` here, instead of the two expectations above  (untested).